ompi/include/mpi.h.in

```diff
@@ -571,6 +571,14 @@ typedef MPI_Win_errhandler_function MPI_Win_errhandler_fn
 #define MPI_MAX_PSET_NAME_LEN    OPAL_MAX_PSET_NAME_LEN /* max pset name len */
 #define MPI_MAX_STRINGTAG_LEN    OPAL_MAX_STRINGTAG_LEN /* max length of string arg to comm from group funcs*/
 
+/*
+ * Constants for C code to access elements in Fortran MPI status array.
+ */
+#define MPI_F_STATUS_SIZE OMPI_FORTRAN_STATUS_SIZE    /* Size of Fortran MPI status array */
+#define MPI_F_SOURCE      0                           /* Index for MPI_SOURCE */       
```

Comment:
@jsquyres I just noticed that this PR introduced trailing whitespace in this line. Maybe you can fix it in your upcoming PR.

Reply:
We should have a CI check for that...  Thanks for noticing; will fix in #10528.